### DISCUSSION TITLE
change name of page to other langwish

### DISCUSSION THREAD

**Discussion by edanTavory** [vercel/community#662] [Show & Tell]
name of page is name of url of that, for seo on other langwish need to change that, how can i do that?

**Comment by swarnava** [comment]
Could you post a screenshot of the url or page you are referring to with more details?

**Reply by edanTavory** [reply]
I need to make routing to page in hebrow language, not translate .
Make the original version routing in hebrow language for seo needs
Thank!
![Uploading image_67171073.JP
![image_67162369](https://user-images.githubusercontent.com/98823191/174065465-6194b85a-5bee-42f3-b43e-ccd741edafc8.JPG)
G…]()
![image_67188737](https://user-images.githubusercontent.com/98823191/174065567-49ad25c8-2d2b-4eca-b0f6-87b7be7a1cc1.JPG)
![image_67215617](https://user-images.githubusercontent.com/98823191/174065776-c62f68f9-dd11-455a-a860-8f697b424e86.JPG)

**Reply by swarnava** [reply]
Here is a steps to get started: https://nextjs.org/docs/advanced-features/i18n-routing. You can also reach out to [Next.js community](https://github.com/vercel/next.js/discussions/) for further help.

**Reply by edanTavory** [reply]
> 

no, that tranlste the web state ,i need to write the url on other langwish
